### ISSUE TITLE
Feature Request: Pull option to not create JS drivers

### ISSUE THREAD

**Issue by mramosbhz** [dittowords/cli#108]
Is there any chance to add an option to the `pull` command to avoid creating the JS drivers (files: index.js and index.d.ts)?

Here in our company, we use our internationalization driver. So everything we need to pull something new from ditto, those 2 files are created crashing the application.

**Comment by azjgard** [comment]
Hi @mramosbhz, thanks for opening.

I think this is a pretty reasonable request. We'll get back to you on this soon!

**Comment by azjgard** [comment]
Hi @mramosbhz, thanks for opening!

Can you please provide more context how your internationalization driver works, especially with regards to how the creation of the `index.js` and `index.d.ts` files are causing your application to crash?

**Comment by mramosbhz** [comment]
Yes, I can.

### i18n driver
We're using a hook based on the `react-localization` package to handle our translations. It's a custom-made one.

### Application crash
Our application is configured to avoid developers defining variables explicitly as `any`.
So, just running the CLI Pull command, it'll create a typescript interface that uses an explicit `any` type:

```ts
/// index.d.ts
...
interface IJSONStructured {
  [key: string]: {
    text: string;
    status?: string;
    notes?: string;
    [property: string]: any;
  };
}
...
```

This will crash the build and the hot compiling of the application. At every new CLI Pull command, it'll be overwritten, so we can not change it.

I can just put another command at the of the npm script to remove the files, like this:
```json
{
  "ditto:pull": "cd src/base-dir-of-copy/$npm_config_project && npx @dittowords/cli pull && rm ditto/index.*",
}
```
But, once again, as I mentioned [here](https://github.com/dittowords/cli/issues/90#issuecomment-2052153398), it'll only work on WSL/Linux.

**Comment by azjgard** [comment]
Thanks for the additional context! This seems like a pretty reasonable ask, especially since there's currently an `any` in that `.d.ts` file which gets generated (which I totally forgot about).

I'll surface this to to the team and we'll get back to you at the beginning of next week!

**Comment by mramosbhz** [comment]
Another problem I found, is that it'll include those 2 files in every single folder project.
We use SonarQube. It will mark those files as `Duplicated Code` throughout our repository.
We're avoiding this nowadays by putting those files on `.gitignore` and not committing them.

**Comment by azjgard** [comment]
@mramosbhz we're going to add an option to disable the generation of `index.js` and `index.d.ts` -- we should have an update for you in the next day or so!

**Comment by mramosbhz** [comment]
Thanks @azjgard !
I'll wait for the new release.

**Comment by azjgard** [comment]
Hi @mramosbhz,

I just published [v4.5.0](https://github.com/dittowords/cli/releases/tag/v4.5.0) which includes a new `disableJsDriver` config option 🚀 when specified, the `index.js` and `index.d.ts` files will not be created.

Feel free to re-open this issue if you encounter any problems with it or it doesn't fit your usecase!
